packages/devtools_server/lib/src/handlers.dart

```diff
@@ -60,7 +74,8 @@ class ServerApi {
   /// Handles all requests.
   ///
   /// To override an API call, pass in a subclass of [ServerApi].
-  static FutureOr<shelf.Response> handle(shelf.Request request, [ServerApi api]) {
+  static FutureOr<shelf.Response> handle(shelf.Request request,
+      [ServerApi api]) {
     api ??= ServerApi();
     switch (request.url.path) {
```

Comment:
@DanTup 

Can you move the sseHandler logic here?

```dart
class ServerApi {
  final SseHandler sseHandler = SseHandler(Uri.parse('/api/sse'))
    ..connections.rest.listen(clients.acceptClient);

  handle(...) {
    switch ... {
    case 'api/sse':
      return sseHandler.handler(request);
  ...
    }
  }
}
```

Reply:
I realize this might mean constructing the ServerApi instance before line 40, depending on what sseHandler wants.